wolfssl/include.am

```diff
@@ -16,10 +16,16 @@ nobase_include_HEADERS+= \
                          wolfssl/certs_test.h \
                          wolfssl/test.h \
                          wolfssl/version.h \
-                         wolfssl/options.h \
                          wolfssl/ocsp.h \
                          wolfssl/crl.h
 
 noinst_HEADERS+= \
                          wolfssl/internal.h
 
+# For distro build don't install options.h.
```

Comment:
No other include header refers to "config.h". Does the file have a purpose beyond showing which build options were enabled, please? Thank you!